src/ChatFeed/MessageForm/Attachment/props.tsx

```diff
@@ -0,0 +1,8 @@
+import { AttachmentStyles } from './styles';
+
+export interface Props {
+  fileName: string;
+  imageUrl?: string;
+  onRemove?: () => void;
+  style?: AttachmentStyles;
```

Comment:
maybe rename to `customStyles` or something like that

Reply:
yep in another issue